\documentclass[../sherrill-Mix_thesis.tex]{subfiles}
\begin{document}
\graphicspath{{im/}{intro/im/}}
\chapter{Introduction}
\section{Impact of HIV}
	In 1981, physicians began to notice a mysterious increase in the occurrences, often clustered in men who had sex with men or intravenous drug users, of Kaposi's sarcoma and pneumocystis pneumonia \citep{Gottlieb1981,Friedman-Kien1981,Hymes1981,Masur1981,Siegal1981,Gottlieb1981a}. 
	
	Kaposi's sarcoma had been a rare cancer in the US found largely in elderly men with Jewish or Mediterranean ancestry \citep{Laor1979}. Kaposi's sarcoma had also been seen in immunocompromised individuals \citep{Klein1974,Myers1974,Kapadia1977} and there were suggestions that it was a virus-associated cancer \citep{Safai1981} although the causative human herpesvirus 8 would not be discovered for another decade \citep{Chang1994,Sitas1999}. 
	
	Pneumocystis pneumonia was known to be caused by infection of the alveoli with the yeast-like fungus \emph{Pneumocystis jirovecii} \citep{Burke1973,Hughes1977}, previously known as \emph{Pneumocystis carinii} \citep{Stringer2009}. Pneumocystis pneumonia was almost exclusively seen patients with suppressed immune systems or immune disorders and rarely if ever in immunocompetent individuals \citep{Hughes1977}.

	The mechanism for this spike of these opportunistic infection was clarified when researchers found severe T cells depletion and decreases in cellular immunity in these patients \citep{Masur1981,Siegal1981,Gottlieb1981a,Gerstoft1982,Masur1982} but the underlying cause remained unclear and eventually termed acquired immune deficiency syndromek. Potential transmissions by transfusion \citep{Ammann1982,Ehrenkranz1982}, injection drug use \citep{Masur1981,Masur1982,Greene1982} and both homosexual \citep{Fannin1982,Gerstoft1982} and heterosexual \citep{Masur1982,Harris1983} contact pointed towards a transmissible agents [[double transmission]]. A virus named lymphadenopathy-associated virus or human T-lymphotropic viruses III and later renamed human immunodefficiency virus was isolated from patient samples in 1983 \citep{Barre-Sinoussi1983,Popovic1984,Levy1984} and soon detected in most immunodeficient patients \citep{Gallo1984,Sarngadharan1984,Safai1984,Levy1984}. The virus was sequenced in 1985 \citep{Wain-Hobson1985}.

	History of the virus and infected population

	%Gallo1983 was contaminated 
	
	%, a rare disorder in the US \citep{Safai1981} that had been seen associated with immunosuppression \citep{Klein1974,Myers1974,Kapadia1977}, and pneumocystis pneumonia, an infection previously reported only in individuals with compromised immune systems \citep{Burke1973,Hughes1977}, in otherwise healthy individuals, often men who have sex with men or intravenous drug users \citep{Gottlieb1981,Friedman-Kien1981,Hymes1981,Masur1981,Siegal1981,Gottlieb1981a}. , a rare tumor now known to be associated with a human herpesvirus \citep{Chang1994,Sitas1999}, and pneumocystis pneumonia, an opportunistic infections,  .  A severe T cells depletion and decrease in cellular immunity was observed in these patients \citep{Masur1981,Siegal1981,Gottlieb1981a,Gerstoft1982,Masur1982}. Potential transmissions by transfusion \citep{Ammann1982,Ehrenkranz1982}, injection drug use \citep{Masur1981,Masur1982,Greene1982} and both homosexual \citep{Fannin1982,Gerstoft1982} and heterosexual \citep{Masur1982,Harris1983} contact were soon observed.

	%The virus was isolated \citep{Barre-Sinoussi1983,Popovic1984,Gallo1984}

	%Sequenced \citep{Wain-Hobson1985}

	%HIV2 \citep{Clavel1986}

	%First HIV antibody test \citep{Safai1984}

	Origin in chimpanzee \citep{Gao1999}
\section{The HIV virus}
\section{Integration and latency}
\section{Host cell interactions}
\section{HIV splicing}
	Driven by a strong selective pressure for genome compactness \citep{Gelinas1986,Herman1987,Shin2000}, HIV and other lentiviruses subvert host cell alternative splicing pathways to allow tight packing of their genetic information. Through weak splice sites and overlapping reading frames (Figure \ref{hivMap}), the virus manages to produce precise quantities of at least nine proteins and polyproteins from its single transcription start site and less than 10 kb genome \citep{Stoltzfus2009}. 

	As such an integral part of the virus life cycle\citep{Kim1989,Pomerantz1990}, alteration of splicing poses a tempting therapeutic target. Inhibition of cellular splicing factors reduces viral reproduction in many genome-wide siRNA screens \citep{Brass2008,Konig2008,Bushman2009} and several members of the spliceosome interact with viral proteins in affinity pulldowns \citep{Jager2012}. Open reading frames in uncharacterized transcripts appear to produce epitopes useful for vaccine development \citep{Bansal2010}. Potential treatments altering viral splicing through small molecule inhibitors \citep{Fukuhara2006,Bakkour2007} and gene therapy \citep{Asparuhova2007,Mandal2010} have restricted viral replication \emph{in vitro}. However without methods to quantify viral splicing or a thorough quantification of splicing under varying conditions, the development of such treatments remains limited. 

	Viral proteins also interact with components of the cellular splicing complex \citep{Tange1996,Berro2006,Jager2012}. These interactions have been reported to change splicing in viral\citep{Berro2006,Bohne2007,Jablonski2010} and cellular transcripts \citep{Kuramitsu2005,Hashizume2007} and raise the possibility that the virus has evolved to alter host splicing. A genome-wide study of changes in cellular splicing during HIV infection would greatly clarify this hypothesis but no such study has been performed. 

	Alternative splicing, the differential inclusion of exons and removal of introns from primary mRNA transcripts, allows rapid evolution of protein segments \citep{Kopelman2005,Xing2005,Su2006} and drastic increases in the number of proteins generated by a single DNA sequence \citep{Watson2005}. Many viruses subvert the splicing machinery of their eukaryotic hosts to modify their viral mRNA \citep{Pollard1998}. 

	In particular, it has previously been reported that HIV utilizes alternative splicing to generate more than 40 mRNA transcripts encoding at least 9 proteins and polyproteins from a genome smaller than 10kb \citep{Purcell1993}. A specific progression of viral transcripts appear in the cytoplasm of the host cell as infection progresses allowing a shift from regulatory protein production in early infection into virion production in late infection \citep{Kim1989,Pomerantz1990,Klotman1991}. Because HIV has only a single transcription start site, these transcriptional changes are driven by alternative splicing \citep{Stoltzfus2009}. 

	 Although it plays such an essential role for the virus, only a single detailed census of viral splicing has been reported \citep{Purcell1993}. Due to limitations in technology, this study was limited to only the most abundant transcripts in lab-adapted HIV strains in cell culture \citep{Purcell1993}. Yet rare transcripts may play an important role in immune response \citep{Bansal2010} and encode unknown proteins \citep{Benko1990}; lab adapted HIV can differ markedly from viruses actually found in patients \citep{Fujita1992};  cell cultures often do not reflect \emph{in vivo} conditions \citep{McAllister1971}; and splicing can vary between humans \citep{Kwan2007,Hull2007} and cell types \citep{Wang2008,Barash2010}. Without a fuller characterization of transcripts under these relevant conditions, many aspects of viral splicing will remain poorly understood.

	Alternative splicing may also play an unappreciated role in HIV-host interactions. Viral proteins interact with the splicing complex \citep{Tange1996,Berro2006,Jager2012} and alter splicing of some cellular transcripts \citep{Kuramitsu2005,Hashizume2007}. Yet, although infection has been shown to cause genome-wide changes in the expression of cellular genes \citep{Vahey2002,Wout2003,Mitchell2003,Rotger2010,Chang2011}, no genome-wide study of cellular alternative splicing during HIV infection has ever been reported. Such a genome-wide study of splicing changes might reveal a distortion of diverse cellular splicing which is adaptively advantageous to the virus. 

	Current sequencing advancements allow a much broader and deeper investigation of viral splicing. Targeted amplification with RainDance droplet PCR offer the potential to reduce size bias inherent in bulk PCR \citep{Tewhey2009}. RNA-seq with Illumina sequencing allows extremely deep sequencing of cellular and viral transcripts with billions of bases of short read sequence \citep{Marioni2008,Morin2008}. Single molecule sequencing with Pacific Biosciences provides reads approaching 20,000 bases \citep{Eid2009,Schaefer2012} that could characterize entire viral transcripts in one continuous read. By combining these technologies, viral and cellular transcripts could be interrogated to an unprecedented level.

	A better understanding of viral splicing and viral effects on host splicing may bring therapeutic benefits. siRNA inhibition of splicing factors reduces HIV replication in many genome-wide screens \citep{Brass2008,Konig2008,Bushman2009}. Alteration of viral splicing through small molecule inhibitor of SR protein kinases\citep{Fukuhara2006} and Splicing Factor 2 \citep{Bakkour2007}, shRNA against spliceosomal U7 snRNP \citep{Asparuhova2007} and expression of modified spliceosomal U1 snRNP \citep{Mandal2010} show treatment potential \emph{in vitro}. In addition, rare uncharacterized HIV transcripts and their encoded proteins appear to produce potent immune response in HIV patients \citep{Bansal2010} thus offering potential targets for vaccine development. Yet without methods to characterize viral RNA and measure the effects of treatments on viral splicing, further development is inhibited. 

	Inclusion and exclusion of a particular stretch of RNA into an mRNA is determined by a balance of RNA secondary structure \citep{Buratti2004,Jablonski2008,Shepard2008}, chromatin structure \citep{Allo2009}, nucleosome positioning \citep{Tilgner2009}, histone marks \citep{Schwartz2009}, previous splicings \citep{Crabb2010}, order of intron removal \citep{Takahara2002,Mata2010} and enhancers \citep{Zahler1993} and suppressors \citep{Smith2000} that bind specific motifs \citep{Ule2006}. Together these factors create a precise controllable splicing code \citep{Barash2010,Xiong2011,Witten2011}.  

	 In HIV, splicing occurs between at least four splice donors and eight splice acceptors \citep{Stoltzfus2009}. Two splice donors, D1 and D4, are relatively strong while the remaining donors and all acceptors are fairly weak \citep{O'Reilly1995}. Several exonic splicing silencers \citep{Amendt1994,Levengood2012} and exon splicing enhancers \citep{Caputi2004,Asang2008} and a single intronic splicing silencer \citep{Tange2001} in the viral genome interact with many human splicing factors, including hnRNPs A1 \citep{Tange2001, Levengood2012} H, F, 2H9, and A2 \citep{Jablonski2008} and SR proteins SRp40\citep{Caputi2004,Tranell2010}, SRp75 \citep{Tranell2010}, ASF/SF2 \citep{Caputi2004} and SC35 \citep{Jablonski2008}, to alter viral splicing \citep{Stoltzfus2006,Stoltzfus2009}.

	Several viral proteins affect mRNA abundances. Rev causes export of unspliced viral mRNA that would otherwise be trapped in the nucleus \citep{Legrain1989} to be exported \citep{Fischer1994,Pollard1998} and may also interact with splicing factors to alter viral splicing \citep{Tange1996}. The HIV protein Tat is best known for its transactivation of viral transcription \citep{Sodroski1985,Jones1994} and triggering apoptosis in uninfected cells \citep{McCloskey1997,Campbell2004} but Tat also appears to independently affect alternative splicing of viral transcripts\citep{Berro2006,Bohne2007,Jablonski2010,Miller2011}. Viral protein Vpr is known to cause cell cycle arrest \citep{Rogel1995} and mediate nuclear import of the viral preintegration complex \citep{Fouchier1998}. Vpr also appears to alter alternative splicing of some cellular transcripts \citep{Kuramitsu2005,Hashizume2007} and interact with the SMN complex \citep{Jager2012}, which assembles spliceosomal snRNP \citep{Gubitz2004}. Although all three of these proteins modify viral splicing, whether they also cause widespread alterations in cellular splicing is unknown.

	Despite the critical role alternative splicing plays in viral replication, no genome-wide studies of lentiviral effects on cellular splicing or  detailed censuses of viral alternative splicing in biologically relevant settings have been published.

	RNA-seq offers a much broader view of alternative splicing than previously possible \citep{Trapnell2010,Rogers2012} but Illumina sequencing has not yet been applied to the study of differential splicing in host RNA of HIV-infected cells. There have been many studies of cellular expression using microarrays \citep{Vahey2002,Wout2003,Mitchell2003,Rotger2010,Miller2011} and Sage \citep{Ryo1999,Lefebvre2011} but only a single study using Illumina RNA-seq and alternative splicing changes were not reported \citep{Chang2011}. Thus a potentially significant aspect of HIV-host interactions remains unknown.

	The most extensive survey of HIV transcripts to date was published in 1993\citep{Purcell1993}. Technology at the time necessitated the use of Northern blots and RNA protection assays \citep{Purcell1993} which can not distinguish multiple similarly sized transcripts or detect rare transcripts. This study also focused on a single lab adapted \hivNL{} strain in HeLa cell culture.

	Many previous studies of viral splicing have used lab-adapted strains of HIV which often differ from patient isolates \citep{Fujita1992} in cell cultures which often differ from primary cells \citep{McAllister1971}. Selection under cell culture conditions may quickly alter splicing patterns to down regulate proteins unneeded \emph{in vitro}.  Characterization of alternative splicing in biologically relevant cell types infected with clinical isolates of HIV are sorely needed.


	%
\section{RNA detection}
	First HIV antibody test \citep{Safai1984,Sarngadharan1984}

\end{document}
