%%% PREAMBLE

\documentclass[11pt]{report} % This is in order to have the chapter feature and page numbers in the same location on even and odd pages. 
% Acceptable font sizes are 10 to 12 pts
\usepackage[pdfusetitle,pdfauthor={Scott Sherrill-Mix},hidelinks]{hyperref}  %[[update with title]]
\usepackage{penn_thesis_style}

% Defining variables to be used throughout the document for personalization
\newcommand{\mylowertitle}{Latency, expression and splicing during infection with HIV} 
\def\myauthor{Scott Sherrill-Mix} \def\myauthorfull{Scott A. Sherrill-Mix}
\def\mysupervisorname{Frederic D. Bushman, Ph.D.}
\def\mysupervisortitle{Professor of Microbiology}
\def\gradchairname{Li-San Wang, Ph.D.}
\def\gradchairtitle{Associate Professor of Pathology and Laboratory Medicine}
\def\mydepartment{Genomics and Computational Biology}
\def\myyear{2015}
\def\signatures{46 pt} % Space to accommodate the signatures, you can fiddle with this as you like
\def\mycommittee{Nancy Zhang, Ph.D. Associate Professor of Statistics \\[\baselineskip] %, University of Pennsylvania
	Yoseph Barash, Ph.D., Assistant Professor of Genetics\\[\baselineskip]%, University of Pennsylvania
	Kristen W. Lynch, Ph.D., Professor of Biochemistry and Biophysics\\[\baselineskip] %, University of Pennsylvania
	Michael Malim, Ph.D., Professor of Infectious Diseases, King's College London
}
\def\mydedication{Dedicated to William Maurer, Gayle Maurer \& Michele Sherrill-Mix}
\def\myacknowledgements{I would like to thank 

Rick

Bushman lab Chris wetlab

Hannah Chervitz and Tiffany Barlow

GCB

committee

collaborators

Xiaofen and Otto

HINT grant and GCB training grant

\ldots

}
\def\myabstract{
	The retrovirus HIV-1 integrates a DNA copy of its genome into the chromosome of its host cell. This provirus can remain latent for years before coopting the host cell to express its viral genes and produce virions. The determinants of latency, the total diversity of viral alternative splicing and the resulting alterations of cellular gene activity remain uncertain. In this thesis, we apply high throughput sequencing techniques to investigate the latency, expression and splicing of HIV-1 and its host cells.

	First, we compare data from five different \textit{in vitro} models of latency with genomic annotations of chromosomal features. The latent phenotype appears to be associated with chromosomal position and certain features within models but not consistently across all five models. Drivers of latency appear to differ among \textit{in vitro} models and perhaps among \textit{in vivo} cell populations.

	Next, we use single-molecule amplification and sequencing to show that the clinical isolate \hivEight{} produces at least 109 different spliced RNAs, including a previously unappreciated $\approximately{}1$ kb class of messages. Viral message populations differed between cell types, longitudinally during infection and among T cells from different human donors. 
	
	We then carry out a deep RNA-Seq analysis of primary human T cells infected with \hivEight{}. Seventeen percent of cellular genes showed altered activity. This data differed from studies of infection in cell lines but showed more parallels with infections of primary cells. We observe an activation of human endogenous retroviruses and retrotransposons and a global trend toward retention of introns after infection. We also quantified $>70$ splice forms of \hivEight{} mRNA and the main types of chimeric HIV-host RNAs.

	Finally, we develop primers optimized to detect HIV-1 RNA using loop mediated isothermal amplification and empirically test 44 new potential primer designs. One primer set targeting the HIV integrase coding region consistently detected HIV-1 subtypes A, B, C, D, and G. This assay might allow quantitative measurements of viral load in resource-limited settings and the detection of infection prior to seroconversion.

	These studies aid in the understanding of HIV expression and latency and its relationship with its host cell. 

%HIV mRNA are expressed from the integrated provirus coding for viral proteins and daughter viral genomes. Need more information on latency, effects on host cell expression, spliceforms of HIC and easier ways to detect HIV infections. In this thesis, we assess latency  investigate HIV splcing   further investigate HIV splicing along with host expression  and work to develop an assay for detecting HIV mRNA in patient samples.

	%HIV infection can be treated effectively with antiretroviral agents, but the persistence of a latent reservoir of integrated proviruses prevents eradication of HIV from infected individuals. The chromosomal environment of integrated proviruses has been proposed to influence HIV latency, but the determinants of transcriptional repression have not been fully clarified, and it is unclear whether the same molecular mechanisms drive latency in different cell culture models. 

%Alternative RNA splicing greatly expands the repertoire of proteins encoded by genomes. Next-generation sequencing (NGS) is attractive for studying alternative splicing because of the efficiency and low cost per base, but short reads typical of NGS only report mRNA fragments containing one or few splice junctions. Here, we used single-molecule amplification and long-read sequencing to study the HIV-1 provirus, which is only 9700 bp in length, but encodes nine major proteins via alternative splicing. Our data showed that the clinical isolate \hivEight{} produces at least 109 different spliced RNAs, including a previously unappreciated $\approximately{}1$ kb class of messages, two of which encode new proteins. HIV-1 message populations differed between cell types, longitudinally during infection, and among T cells from different human donors. These findings open a new window on a little studied aspect of HIV-1 replication, suggest therapeutic opportunities and provide advanced tools for the study of alternative splicing. 

	%Background: HIV infection has been reported to alter cellular gene activity, but published studies have commonly assayed transformed cell lines and lab-adapted HIV strains, yielding inconsistent results. Here we carried out a deep RNA-Seq analysis of primary human T cells infected with the low passage HIV isolate \hivEight{}. 

	%Results: 

	%Conclusions: Analysis of RNA abundance after infection of primary T cells with the low passage \hivEight{} isolate disclosed multiple novel features of HIV-host interactions, notably intron retention and induction of transcription of distinctive retrotransposons and endogenous retroviruses.

	%Diagnostic methods for detecting and quantifying HIV RNA have been improving, but efficient methods for point-of-care analysis are still needed, particularly for applications in resource-limited settings. Detection based on reverse-transcription loop-mediated isothermal amplification (RT-LAMP) is particularly useful for this, because when combined with fluorescence-based DNA detection, RT-LAMP can be implemented with minimal equipment and expense. Assays have been developed to detect HIV RNA with RT-LAMP, but existing methods detect only a limited subset of HIV subtypes. Here we report a bioinformatic study to develop optimized primers, followed by empirical testing of 44 new primer designs. One primer set (ACeIN-26), targeting the HIV integrase coding region, consistently detected subtypes A, B, C, D, and G. The assay was sensitive to at least 5000 copies per reaction for subtypes A, B, C, D, and G, with Z-factors of above 0.69 (detection of the minor subtype F was found to be unreliable). There are already rapid and efficient assays available for detecting HIV infection in a binary yes/no format, but the rapid RT-LAMP assay described here has additional uses, including 1) tracking response to medication by comparing longitudinal values for a subject, 2) detecting of infection in neonates unimpeded by the presence of maternal antibody, and 3) detecting infection prior to seroconversion.



%No more than 350 words. It is normally a single paragraph, consists of four parts: the statement of the problem; the procedure and methods used to investigate the problem; the results of the investigation; and the conclusions. The abstract is published online by ProQuest in ``Dissertation Abstracts International'', providing information to interested readers about the general content of the dissertation.
}




%abbreviations
\newcommand{\cdFour}{CD4$^+$}
\newcommand{\hivEight}{HIV$_{89.6}$}
\newcommand{\hivLai}{HIV$_{\textrm{LAI}}$}
\newcommand{\threePrime}{3\ensuremath{'}}
\newcommand{\fivePrime}{5\ensuremath{'}}
\newcommand{\degree}{\ensuremath{^\circ}}
\newcommand{\uM}{\ensuremath{\mu}M}
\newcommand{\uL}{\ensuremath{\mu}L}
\newcommand{\ug}{\ensuremath{\mu}g}
\newcommand{\approximately}{\mathord{\sim}}
\newcommand{\hivNL}{HIV$_{\text{NL4-3}}$}
\newcommand{\gag}{\emph{gag}}
\newcommand{\pol}{\emph{pol}}
\newcommand{\tat}{\emph{tat}}
\newcommand{\rev}{\emph{rev}}
\newcommand{\nef}{\emph{nef}}
\newcommand{\vpr}{\emph{vpr}}
\newcommand{\vif}{\emph{vif}}
\newcommand{\vpu}{\emph{vpu}}
\newcommand{\env}{\emph{env}}


%%% BEGIN DOCUMENT

\begin{document}
\input{headMatter.tex}

\subfile{intro/intro}

\subfile{rnaSeq/rnaSeq}

\subfile{latency/latency}

\subfile{lamp/lamp}

\subfile{pacBio/pacBio}

\subfile{conclusion/conclusion}

\end{mainf}

%% APPENDICES (OPTIONAL)
\appendix

\begin{appendixf}

%\addtocontents{toc}{\protect\setcounter{tocdepth}{-1}} % This is to fix how appendices are shown in ToC
%\chapter{Appendix}
%\addtocontents{toc}{\protect\setcounter{tocdepth}{1}} % This is to bring things back to normal
%\addcontentsline{toc}{chapter}{APPENDIX} % You have to write here everything you want the ToC to show including \thechapter if you want numbering
%\addtocontents{toc}{\protect\setcounter{tocdepth}{-1}} % This is so sections in the appendix are not shown in the ToC


\subfile{appendices/latencyReport/AdditionalFile4.tex}

%\addtocontents{toc}{\protect\setcounter{tocdepth}{1}} % This is to bring things back to normal
\end{appendixf}

%% BIBLIOGRAPHY


\begin{bibliof}

\bibliographystyle{unsrtnat}
\bibliography{refs} % This is the filename of the bibtex bibliography file (it has to be in the same directory as the main LaTeX file)

\end{bibliof}
%% INDEX (OPTIONAL) - I do not really know how to code this, sorry

\end{document}
