%%% PREAMBLE

\documentclass[11pt]{report} % This is in order to have the chapter feature and page numbers in the same location on even and odd pages. 
% Acceptable font sizes are 10 to 12 pts
\usepackage[pdfusetitle,pdfauthor={Scott Sherrill-Mix},hidelinks]{hyperref}  %[[update with title]]
\usepackage{penn_thesis_style}

% Defining variables to be used throughout the document for personalization
\newcommand{\mylowertitle}{Latency, expression and splicing during infection with HIV} 
\def\myauthor{Scott Sherrill-Mix} \def\myauthorfull{Scott A. Sherrill-Mix}
\def\mysupervisorname{Frederic D. Bushman, Ph.D.}
\def\mysupervisortitle{Professor of Microbiology}
\def\gradchairname{Li-San Wang, Ph.D.}
\def\gradchairtitle{Associate Professor of Pathology and Laboratory Medicine}
\def\mydepartment{Genomics and Computational Biology}
\def\myyear{2015}
\def\signatures{46 pt} % Space to accommodate the signatures, you can fiddle with this as you like
\def\mycommittee{Nancy Zhang, Ph.D. Associate Professor of Statistics \\[\baselineskip] %, University of Pennsylvania
	Yoseph Barash, Ph.D., Assistant Professor of Genetics\\[\baselineskip]%, University of Pennsylvania
	Kristen Lynch, Ph.D., Professor of Biochemistry and Biophysics\\[\baselineskip] %, University of Pennsylvania
	Michael Malim, Ph.D., Professor of Infectious Diseases, King's College London
}
\def\mydedication{Dedicated to William Maurer, Gayle Maurer \& Michele Sherrill-Mix}
\def\myacknowledgements{I would like to thank 

Rick

Bushman lab Chris wetlab
collaborators

Friends in the Bushman lab and classmates in GCB.

My committee---Nancy Zhang, Yosephy Barash, Kristen Lynch and Michael Malim---have provided . Many faculty of GCB mentoring and teaching. Hannah Chervitz and Tiffany Barlow for managing everything and helping manage the layers of bureacracy. Funding from the HIV Immune Networks Team (HINT) consortium P01 AI090935 and NRSA computational genomics training grant T32 HG000046. 

Xiaofen and Otto


\ldots

}
\def\myabstract{ %%ABSTRACT_START%%
	To replicate, the human immunodeficiency virus (HIV-1) integrates a DNA copy of its RNA genome into host cell chromosomes. This provirus can commandeer host cell machinery to express viral genes and produce virions or alternatively remain latent for years. Here, we applied high-throughput sequencing techniques to investigate the latency, expression and splicing of HIV-1 and its effects on host cells.

	First, we investigated the association between the location of proviral integration and latency. We compared integration site data for latent and active provirus from five cell culture models to genomic and epigenetic features. Latency was associated with chromosomal position within models but mechanisms of latency appear to differ between cell culture models. These differences suggest that cell culture models may not reflect latency in patients and that an easily-targetable mechanism for the induction of latent HIV appears unlikely.

	Next, we carried out two studies to explore mRNA populations during HIV infection. We used single-molecule amplification and sequencing to show that the clinical isolate \hivEight{} produces at least 109 different spliced mRNAs. Viral message populations differed between cell types, between human donors and longitudinally during infection. We then sequenced mRNA from control and \hivEight{}-infected primary human T cells. Over 17 percent of cellular genes showed altered activity associated with infection. These gene expression changes differed from HIV infection in cell lines but paralleled infections in primary cells. Infection with \hivEight{} was associated with increased intron retention in cellular genes and increased abundance of RNA from human endogenous retroviruses. We also quantified the frequency and location of chimeric HIV-host RNAs. These two studies together provided a full accounting of more than 70 spliceforms of \hivEight{} and revealed novel expression and splicing changes in both HIV and host mRNA.

	Finally, we developed improved methods for assaying HIV RNA using loop-mediated isothermal amplification. We developed methods based on primers targeted to regions of the HIV-1 genome conserved across subtypes. Combined with lab on a chip technology, these techniques allows quantitative measurements of viral load in a point of care device targeted to resource-limited settings.

%No more than 350 words. It is normally a single paragraph, consists of four parts: the statement of the problem; the procedure and methods used to investigate the problem; the results of the investigation; and the conclusions. The abstract is published online by ProQuest in ``Dissertation Abstracts International'', providing information to interested readers about the general content of the dissertation.
} %%ABSTRACT_END%%




%abbreviations
\newcommand{\cdFour}{CD4$^+$}
\newcommand{\hivEight}{HIV$_{89.6}$}
\newcommand{\hivLai}{HIV$_{\textrm{LAI}}$}
\newcommand{\threePrime}{3\ensuremath{'}}
\newcommand{\fivePrime}{5\ensuremath{'}}
\newcommand{\degree}{\ensuremath{^\circ}}
\newcommand{\uM}{\ensuremath{\mu}M}
\newcommand{\uL}{\ensuremath{\mu}L}
\newcommand{\ug}{\ensuremath{\mu}g}
\newcommand{\approximately}{\mathord{\sim}}
\newcommand{\hivNL}{HIV$_{\text{NL4-3}}$}
\newcommand{\gag}{\emph{gag}}
\newcommand{\pol}{\emph{pol}}
\newcommand{\tat}{\emph{tat}}
\newcommand{\rev}{\emph{rev}}
\newcommand{\nef}{\emph{nef}}
\newcommand{\vpr}{\emph{vpr}}
\newcommand{\vif}{\emph{vif}}
\newcommand{\vpu}{\emph{vpu}}
\newcommand{\env}{\emph{env}}


%%% BEGIN DOCUMENT

\begin{document}
\input{headMatter.tex}

\subfile{intro/intro}

\subfile{rnaSeq/rnaSeq}

\subfile{latency/latency}

\subfile{lamp/lamp}

\subfile{pacBio/pacBio}

\subfile{conclusion/conclusion}

\end{mainf}

%% APPENDICES (OPTIONAL)
\appendix

\begin{appendixf}

%\addtocontents{toc}{\protect\setcounter{tocdepth}{-1}} % This is to fix how appendices are shown in ToC
%\chapter{Appendix}
%\addtocontents{toc}{\protect\setcounter{tocdepth}{1}} % This is to bring things back to normal
%\addcontentsline{toc}{chapter}{APPENDIX} % You have to write here everything you want the ToC to show including \thechapter if you want numbering
%\addtocontents{toc}{\protect\setcounter{tocdepth}{-1}} % This is so sections in the appendix are not shown in the ToC


\subfile{appendices/latencyReport/AdditionalFile4.tex}

%\addtocontents{toc}{\protect\setcounter{tocdepth}{1}} % This is to bring things back to normal
\end{appendixf}

%% BIBLIOGRAPHY


\begin{bibliof}

\bibliographystyle{unsrtnat}
\bibliography{refs} % This is the filename of the bibtex bibliography file (it has to be in the same directory as the main LaTeX file)

\end{bibliof}
%% INDEX (OPTIONAL) - I do not really know how to code this, sorry

\end{document}
