%%% PREAMBLE

\documentclass[11pt]{report} % This is in order to have the chapter feature and page numbers in the same location on even and odd pages. 
% Acceptable font sizes are 10 to 12 pts
\usepackage{penn_thesis_style}


\newif\ifdraft\drafttrue %change to [[draftfalse when not a draft ]]
\ifx\draft\defined
	\ifdraft
		\overfullrule=15pt
		\usepackage{lineno}
		%\modulolinenumbers[5]
		\definecolor{lineCol}{gray}{.7}
		\renewcommand\linenumberfont{\color{lineCol}\normalfont\tiny\sffamily}
		\linenumbers
	\fi
\fi
% Defining variables to be used throughout the document for personalization
\newcommand{\mylowertitle}{Latency, expression and splicing during infection with HIV} 
\def\myauthor{Scott Sherrill-Mix} \def\myauthorfull{Scott A. Sherrill-Mix}
\def\mysupervisorname{Frederic D. Bushman, Ph.D.}
\def\mysupervisortitle{Professor of Microbiology}
\def\gradchairname{Li-San Wang, Ph.D.}
\def\gradchairtitle{Associate Professor of Pathology and Laboratory Medicine}
\def\mydepartment{Genomics and Computational Biology}
\def\myyear{2015}
\def\signatures{46 pt} % Space to accommodate the signatures, you can fiddle with this as you like
\def\mycommittee{Nancy Zhang, Ph.D. Associate Professor of Statistics \\[\baselineskip] %, University of Pennsylvania
	Yoseph Barash, Ph.D., Assistant Professor of Genetics\\[\baselineskip]%, University of Pennsylvania
	Kristen Lynch, Ph.D., Professor of Biochemistry and Biophysics\\[\baselineskip] %, University of Pennsylvania
	Michael Malim, Ph.D., Professor of Infectious Diseases, King's College London
}
\def\mydedication{Dedicated to William Maurer, Gayle Maurer \& Michele Sherrill-Mix}
\def\myacknowledgements{I would like to thank 

Rick Bushman

Christian Hoffmann wet lab

collaborators

Bushman lab Rithun Mukherjee, Karen Ocwieja, Nirav Malani, Troy Brady, Young Hwang, Brendan Kelly, Kyle Bittinger, Rebecca Custers-Allen, Serena Dollive, Frances Male, Jacque Young, Rohini Sinha, Sam Minot, Aubrey Bailey, Christopher Nobles, Stephanie Grunberg, Vesa Turkki, Anatoly Dryga, Eric Sherman, Greg Peterfreund, Yinghua Wu, Alice Laughlin, Sesh Sundararaman, Alexandra Bryson, Christel Chehoud, Erik Clarke, Arwa Abbas

%Friends in the Bushman lab and classmates in GCB. Fan Paul Nate Yichii

My committee---Nancy Zhang, Yoseph Barash, Kristen Lynch and Michael Malim---have provided guidance and encouragement. Many faculty of GCB mentoring and teaching. Hannah Chervitz, Tiffany Barlow, Mali Skotheim, Caitlin Greig and Laurie Zimmerman for managing everything and helping manage the layers of bureacracy. Funding from the HIV Immune Networks Team (HINT) consortium P01 AI090935 and NRSA computational genomics training grant T32 HG000046. 

Ram Myers and Mike James for great previous mentoring.

Xiaofen and Otto

\ldots

}
\def\myabstract{ %%ABSTRACT_START%%
	Over 35 million people are living with human immunodeficiency virus (HIV-1). The mechanisms causing integrated provirus to become latent, the diversity of spliced viral transcripts and the cellular response to infection are not fully characterized and hinder the eradication of HIV-1. We applied high-throughput sequencing to investigate the effects of host chromatin on proviral latency and variation of expression and splicing in both the host and virus during infection.

	To evaluate the link between host chromatin and proviral latency, we compared genomic and epigenetic features to HIV-1 integration site data for latent and active provirus from five cell culture models. Latency was associated with chromosomal position within individual models. However, no shared mechanisms of latency were observed between cell culture models. These differences suggest that cell culture models may not completely reflect latency in patients.

	We carried out two studies to explore mRNA populations during HIV infection. Single-molecule amplification and sequencing revealed that the clinical isolate \hivEight{} produces at least 109 different spliced mRNAs. Viral message populations differed between cell types, between human donors and longitudinally during infection. We then sequenced mRNA from control and \hivEight{}-infected primary human T cells. Over 17 percent of cellular genes showed altered activity associated with infection. These gene expression patterns differed from HIV infection in cell lines but paralleled infections in primary cells. Infection with \hivEight{} increased intron retention in cellular genes and abundance of RNA from human endogenous retroviruses. We also quantified the frequency and location of chimeric HIV-host RNAs. These two studies together provided a detailed accounting of both \hivEight{} and host expression and alternative splicing.

	A more cost-effective method of detecting viral load would aid patients with poor access to healthcare. We developed improved methods for assaying HIV-1 RNA using loop-mediated isothermal amplification based on primers targeting regions of the HIV-1 genome conserved across subtypes. Combined with lab-on-a-chip technology, these techniques allow quantitative measurements of viral load in a point-of-care device targeted to resource-limited settings.

	This work disclosed novel HIV-host interactions and developed techniques and knowledge that will aid in the study and management of HIV-1 infection.

%No more than 350 words. It is normally a single paragraph, consists of four parts: the statement of the problem; the procedure and methods used to investigate the problem; the results of the investigation; and the conclusions. The abstract is published online by ProQuest in ``Dissertation Abstracts International'', providing information to interested readers about the general content of the dissertation.
} %%ABSTRACT_END%%




%abbreviations
\newcommand{\cdFour}{CD4$^+$}
\newcommand{\hivEight}{HIV$_{89.6}$}
\newcommand{\hivLai}{HIV$_{\textrm{LAI}}$}
\newcommand{\threePrime}{3\ensuremath{'}}
\newcommand{\fivePrime}{5\ensuremath{'}}
\newcommand{\degree}{\ensuremath{^\circ}}
\newcommand{\uM}{\ensuremath{\mu}M}
\newcommand{\uL}{\ensuremath{\mu}L}
\newcommand{\ug}{\ensuremath{\mu}g}
\newcommand{\approximately}{\mathord{\sim}}
\newcommand{\hivNL}{HIV$_{\text{NL4-3}}$}
\newcommand{\gag}{\emph{gag}}
\newcommand{\pol}{\emph{pol}}
\newcommand{\tat}{\emph{tat}}
\newcommand{\rev}{\emph{rev}}
\newcommand{\nef}{\emph{nef}}
\newcommand{\vpr}{\emph{vpr}}
\newcommand{\vif}{\emph{vif}}
\newcommand{\vpu}{\emph{vpu}}
\newcommand{\env}{\emph{env}}


%%% BEGIN DOCUMENT

\begin{document}
\makePreliminary{}

\begin{mainMatter}

\subfile{intro/intro}

\subfile{latency/latency}

\subfile{pacBio/pacBio}

\subfile{rnaSeq/rnaSeq}

\subfile{lamp/lamp}

\subfile{conclusion/conclusion}

\end{mainMatter}

\ifx\draft\defined
	\ifdraft
		\nolinenumbers
	\fi
\fi

%% APPENDICES (OPTIONAL)
\begin{appendixf}

%\addtocontents{toc}{\protect\setcounter{tocdepth}{-1}} % This is to fix how appendices are shown in ToC
%\chapter{Appendix}
%\addtocontents{toc}{\protect\setcounter{tocdepth}{1}} % This is to bring things back to normal
%\addcontentsline{toc}{chapter}{APPENDIX} % You have to write here everything you want the ToC to show including \thechapter if you want numbering
%\addtocontents{toc}{\protect\setcounter{tocdepth}{-1}} % This is so sections in the appendix are not shown in the ToC


\subfile{appendices/pacbioReport/pacbioReport.tex}

\subfile{appendices/latencyReport/AdditionalFile4.tex}

%\addtocontents{toc}{\protect\setcounter{tocdepth}{1}} % This is to bring things back to normal
\end{appendixf}

%% BIBLIOGRAPHY


\begin{bibliof}
\bibliographystyle{unsrtnat2}
\bibliography{refs} % This is the filename of the bibtex bibliography file (it has to be in the same directory as the main LaTeX file)
\end{bibliof}
%% INDEX (OPTIONAL) - I do not really know how to code this, sorry

\end{document}
